cmd/configure.ac

```diff
@@ -178,5 +178,22 @@ AC_PATH_PROG([HAVE_VALGRIND],[valgrind])
 AM_CONDITIONAL([HAVE_VALGRIND], [test "x${HAVE_VALGRIND}" != "x"])
 AS_IF([test "x$HAVE_VALGRIND" = "x"], [AC_MSG_WARN(["cannot find the valgrind tool, will not run unit tests through valgrind"])])
 
+# This knob controls if builds snap-confine are done in a pretty unusual way.
+# The goal is to link dynamically to libc and anything that talks to remote
+# things that we cannot control and that have a volatile protocol, specifically
+# this is udev.
+#
```

Comment:
now I'm confused, why do you need the full general option if we have single ones?  anyway the doc is still need improving and if it's the case explain that this needs to be combined with the specific lib options